go/api/connection_management_commands_test.go

```diff
@@ -41,3 +41,27 @@ func ExampleGlideClient_Echo() {
 
 	// Output: {Hello World false}
 }
+
+func ExampleGlideClient_ClientSetName() {
+	var client *GlideClient = getExampleGlideClient() // example helper function
+	result, err := client.ClientSetName("ConnectionName")
+	if err != nil {
+		fmt.Println("Glide example failed with an error: ", err)
+	}
+	fmt.Println(result)
+
+	// Output:
+	// OK
+}
+
+func ExampleGlideClient_ClientGetName() {
+	var client *GlideClient = getExampleGlideClient() // example helper function
+	result, err := client.ClientGetName()
+	if err != nil {
+		fmt.Println("Glide example failed with an error: ", err)
+	}
+	fmt.Println(result)
+
+	// Output:
+	// ConnectionName
```

Comment:
Order of tests is not guaranteed. Furthermore, each test has new client with new connection.
Expect a default name or set a name prior to getting it